src/encoded/schemas/changelogs/functional_characterization_series.md

```diff
@@ -13,6 +13,7 @@
 * Added a calculated boolean property *datapoint* 
 * Modified *assay_title* calculated properties to not include Controls, unless the Series contains only Control experiments
 * Removed *control_type* calculated property
+* Added *biosamples* calculated property and removed *replicates* calculated property
```

Comment:
I think that this needs to be a new version.  By removing replicate we are breaking code the is in existence for someone.

Reply:
upgraded to 4